scripts/post_release_prep.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -x
+
+# To be executed after release has been deployed to prod
+# pre-requisites: install github CLI
+# - github documentation: https://github.com/cli/cli#installation
```

Comment:
Might also want to check if `gh` exist on the shell env before executing the script since there's a `git push` before `gh` execution making re-runs of this script to be 
non-idempotent (might have side-effects).

Although, this probably should be checked on the release script too.

```
which gh > /dev/null

if [[ $? -gt 0 ]]
then
    echo "github CLI not in installed. See https://github.com/cli/cli#installation"
    # or send them to https://www.google.com/search?q=github+cli
    exit 1
fi
```